### DISCUSSION TITLE
Failed Import - Launch Id, Skip failing files

### DISCUSSION THREAD

**Discussion by darren-pimenta-dotmatics** [reportportal/reportportal#2192] [Q&A]
We are looking to use the API to Import Junit results from a pipeline.
I am seeing this response on occasion:
`{"errorCode":40035,"message":"Error while importing the file. 'Error during parsing the xml file: 'Content is not allowed in prolog.''"}`
Is it possible to skip files that are failing to import?

I can see that a launch is created, so would it be possible to provide the launch id in the response as well?
(I may be missing something but there is no indication in the launch that the import failed to complete successfully)

**Comment by FabianK-Dev** [comment]
Having a similar error:
```
{"errorCode":40035,"message":"Error while importing the file. 'Cannot invoke \"String.isEmpty()\" because \"errType\" is null'"}
```
Did you by any chance manage to fix or workaround it? Thanks for any help.
